source/quickguides/generate_quick_guides.py

```diff
@@ -10,6 +10,14 @@
         'html_path': '/var/www/acme/acme-diags/<username>/',
         'web_address': 'https://acme-viewer.llnl.gov/<username>/',
     },
+    'chrysalis': {
```

Comment:
I'm wondering if we should replace `'Chrysalis'` with `'LCRC'` (and similarly `'Cori'` with `'NERSC'`). I believe all these paths are the same regardless if the user is on Anvil or Chrysalis.

Reply:
I agree, 'LCRC' is clearer since it can be used on Anvil and Chrysalis.